data/json/mapgen/park.json

```diff
@@ -336,198 +248,160 @@
+      "place_item": [
+        { "item": "bag_plastic", "x": 10, "y": 11, "repeat": [ 10, 50 ] },
+        { "item": "feces_dog", "x": [ 3, 20 ], "y": [ 3, 20 ], "chance": 200 },
+        { "item": "feces_dog", "x": 3, "y": 10, "repeat": [ 1, 10 ] },
+        { "item": "feces_dog", "x": 10, "y": 3, "repeat": [ 1, 10 ] },
+        { "item": "feces_dog", "x": 20, "y": 13, "repeat": [ 1, 10 ] },
+        { "item": "feces_dog", "x": 20, "y": 10, "repeat": [ 1, 10 ] },
+        { "item": "feces_dog", "x": 11, "y": 12, "repeat": [ 5, 20 ] }
+      ],
+      "monsters": { ",": { "monster": "GROUP_PARK_DOG", "chance": 30 }, " ": { "monster": "GROUP_PARK_DOG", "chance": 60 } }
     }
   },
   {
     "method": "json",
+    "om_terrain": "park",
+    "//": "Picnic Park",
+    "type": "mapgen",
+    "weight": 300,
     "object": {
-      "furniture": {
-        "#": "f_bench",
-        "&": "f_trashcan",
-        ".": "f_null",
-        "0": "f_boulder_medium",
-        "4": "f_null",
-        "7": "f_null",
-        "C": "f_null",
-        "S": "f_null",
-        "T": "f_null",
-        "W": "f_null",
-        "_": "f_null",
-        "b": "f_brazier",
-        "c": "f_cattails",
-        "d": "f_null",
-        "e": "f_null",
-        "f": "f_dandelion",
-        "g": "f_null",
-        "h": "f_null",
-        "i": "f_cattails",
-        "m": "f_null",
-        "p": "f_null",
-        "s": "f_null",
-        "t": "f_table",
-        "u": "f_null",
-        "w": "f_null"
-      },
+      "fill_ter": "t_region_groundcover_urban",
+      "rows": [
+        "'ppppp''''''''''''''''''",
+        "'ppppp''''''''''''''''''",
+        "'pppppppppppppppp'''''''",
+        "'pppppppppppppppp'''''''",
+        "'PppppPppppPppppP''''u''",
+        "'PppppPppppPppppP'''''''",
+        "'PppppPppppPppppP'''''''",
+        "'PppppPppppPppppP'''''4'",
+        "'PppppPppppPppppP'''''''",
+        "'''''''''''ss'''''''''''",
+        "''''''&ssssssssss&''''e'",
+        "''''''sBBBBssBBBBs''''C'",
+        "'''''YsLLLLssLLLLsY'''''",
+        "''''''sBBBBssBBBBs''4'4'",
+        "'''t''ssssssssssssWWu'''",
+        "''''$$'''''''t'''WwwW'u'",
+        "'''$g$''''''''''Wwwwwi4'",
+        "''''''''''''''''Wwwwwi'4",
+        "''''''mm'h'''''WwwwwiI''",
+        "'''4''''''''''Wwwwwwi''u",
+        "''''''''''4'''WwwwwiI'u4",
+        "'4'''''4'''''''WWWi4'4''",
+        "'''u4'''4''0u'4u'''''u4u",
+        "''''''u''''u4''''u4'''''"
+      ],
+      "palettes": [ "park_scenic_palette" ],
       "place_items": [
         { "chance": 8, "item": "dogfight", "x": 2, "y": 12 },
         { "chance": 30, "item": "trash", "x": 6, "y": 10 },
         { "chance": 10, "item": "child_items", "x": 7, "y": 18 },
         { "chance": 3, "item": "creepy", "x": 22, "y": 21 },
         { "chance": 10, "item": "trash_forest", "x": 10, "y": 23 },
-        { "chance": 20, "item": "trash_forest", "x": 18, "y": 22 },
+        { "chance": 500, "item": "trash_forest", "x": 18, "y": 22 },
```

Comment:
Values that large are not valid here. Valid values are in the range 1 to 100.

Reply:
Reported in #36637

Reply:
@I-am-Erk Should it be `50` (like in 50%) opposed to `500` (1 in 500)?

Reply:
Yes, that looks like an extra zero typo